docs/tutorial/README.md

```diff
@@ -6,5 +6,11 @@ tutorials.
 You may already have worked your way through the [getting
 started](?id=getting-started) one.
 
-For a much more in depth encounter with qooxdoo, have a look at the
-comprehensive [tweets app tutorial](twitter/).
+For a much more in depth encounter with qooxdoo, we have some comprehensive tutorials: 
+
+- **qooxdoo tutorial app** ([App](apps://apiviewer) [Code](https://github.com/qooxdoo/qxl.apiviewer))
+
+- [**tweets app tutorial**](twitter/) ([App](apps://apiviewer) [Code](https://github.com/qooxdoo/qxl.apiviewer))
```

Comment:
The twitter app doesn't, and without a lot of work, can't work. We should not be advertising it.

Reply:
Then we should remove it from the docs too.

Reply:
Did you try to migrate the twitter app? Migration was something on my list. After migrating the feedreader...

Reply:
Is it the lack of the identi.ca backend only or more fundamental problems?

Reply:
The lack of identi.ca had been noticed at some point, and someone had attempted to work around it with a local kludge. I don't recall what that kludge was, but when I looked at the code, I didn't see how it could work without a bunch of change. At the time, I think my suggestion was that we should have a different tutorial...

Reply:
i removed twitter.